packages/runtime/src/extensibility/facades/consumption/OpenId4VcFacade.ts

```diff
@@ -1,47 +1,47 @@
 } from "../../../useCases";
 
 export class OpenId4VcFacade {
     public constructor(
         @Inject private readonly resolveCredentialOfferUseCase: ResolveCredentialOfferUseCase,
-        @Inject private readonly fetchOfferUseCase: FetchCredentialOfferUseCase,
-        @Inject private readonly resolveFetchedOfferUseCase: ResolveFetchedCredentialOfferUseCase,
+        @Inject private readonly resolveAndAcceptCredentialOfferUseCase: ResolveAndAcceptCredentialOfferUseCase,
+        @Inject private readonly acceptCredentialOfferUseCase: AcceptCredentialOfferUseCase,
         @Inject private readonly resolveAuthorizationRequestUseCase: ResolveAuthorizationRequestUseCase,
         @Inject private readonly acceptAuthorizationRequestUseCase: AcceptAuthorizationRequestUseCase,
         @Inject private readonly getVerifiableCredentialsUseCase: GetVerifiableCredentialsUseCase
     ) {}
 
-    public async resolveCredentialOffer(request: ResolveCredentialOfferRequest): Promise<Result<VerifiableCredentialDTO, ApplicationError>> {
-        return await this.resolveCredentialOfferUseCase.execute(request);
+    public async resolveAndAcceptCredentialOffer(request: ResolveAndAcceptCredentialOfferRequest): Promise<Result<VerifiableCredentialDTO, ApplicationError>> {
+        return await this.resolveAndAcceptCredentialOfferUseCase.execute(request);
     }
 
-    public async fetchCredentialOffer(request: FetchCredentialOfferRequest): Promise<Result<FetchedCredentialOfferDTO, ApplicationError>> {
-        return await this.fetchOfferUseCase.execute(request);
+    public async resolveCredentialOffer(request: ResolveCredentialOfferRequest): Promise<Result<ResolvedCredentialOfferDTO, ApplicationError>> {
+        return await this.resolveCredentialOfferUseCase.execute(request);
     }
 
-    public async resolveFetchedCredentialOffer(request: FetchedCredentialOfferRequest): Promise<Result<VerifiableCredentialDTO, ApplicationError>> {
-        return await this.resolveFetchedOfferUseCase.execute(request);
+    public async acceptCredentialOffer(request: AcceptCredentialOfferRequest): Promise<Result<VerifiableCredentialDTO, ApplicationError>> {
+        return await this.acceptCredentialOfferUseCase.execute(request);
     }
 
-    public async resolveAuthorizationRequest(request: ResolveAuthorizationRequestRequest): Promise<Result<FetchedAuthorizationRequestDTO, ApplicationError>> {
+    public async resolveAuthorizationRequest(request: ResolveAuthorizationRequestRequest): Promise<Result<ResolvedAuthorizationRequestDTO, ApplicationError>> {
         return await this.resolveAuthorizationRequestUseCase.execute(request);
     }
 
-    public async acceptAuthorizationRequest(request: AcceptAuthorizationRequestRequest): Promise<Result<AcceptAuthorizationRequestDTO, ApplicationError>> {
+    public async acceptAuthorizationRequest(request: AcceptAuthorizationRequestRequest): Promise<Result<Oid4VpVerificationResultDTO, ApplicationError>> {
```

Comment:
Should it really be called `Oid4VpVerificationResultDTO`? It's the first time
* the name contains `Oid4V`
* we talk about Verification (so far it's called `Authorization`)

=> Shouldn't it rather be called `AcceptAuthorizationRequestResult` or `AuthorizationAcceptanceResult`?

Reply:
(I just updated my comment)